src/Microsoft.AspNetCore.Razor.LanguageServer/RazorCompletionEndpoint.cs

```diff
@@ -125,7 +139,7 @@ public CompletionRegistrationOptions GetRegistrationOptions()
             {
                 DocumentSelector = RazorDefaults.Selector,
                 ResolveProvider = true,
-                TriggerCharacters = new Container<string>("@"),
+                TriggerCharacters = new Container<string>("@", "<"),
```

Comment:
Ooh interesting. Will this be triggered for any `<` in the document even in CSharp context?

Reply:
Sadly yes. Therefore a C# completion list will come up; however, no TagHelper or Html completions will be available because it'll be an invalid Html/TagHelper context.

Reply:
Is it something omnisharp has to support if we want to do it right?

Reply:
Not entirely, we'll have to special case the scenario, I filed an issue: https://github.com/aspnet/Razor.VSCode/issues/295